guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +33,39 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
+* All labels are composed of entirely lowercase characters regardless of actual label capitalization.
+* Pushes follow one of two formats:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push <image> example.com/<organization_label>/<product_label>/<repository_name>[:tag]
+# podman push <image> example.com/id/<organization_id>/<product_id>/<repository_name>[:tag]
+----
+
+* A complete push looks like the following:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# podman push d2c94e258dcb example.com/default_organization/test_product/hello-world:latest
+Getting image source signatures
+Copying blob ac28800ec8bb done   |
+Copying config d2c94e258d done   |
+Writing manifest to image destination
+----
+
+.Limitations on container registries
+* A repository in Katello is created once the content is pushed.
+* If the organization or product does not exist, Katello sends an error telling the user to ensure the proper entities are created first.
+* Subsequent pushes to the same repository updates the contents of that repository.
+* Content is pulled as soon as it is pushed.
```

Comment:
This is curious. The procedure mentions pulling container images manually. Does this mean that when you push a container image, that image gets automatically pulled afterwards?

I don't know what to do about this sentence, but I felt the need to mention that it confused me in the context of the whole module :shrug:

Reply:
@ianballou Any chance you can provide some clarity here?

Reply:
I added my thoughts on this to a review, let me know if there are lingering questions.